### DISCUSSION TITLE
Feature Request: iOS - Open Conversation From Notification

### DISCUSSION THREAD

**Discussion by hnt3b453** [balzack/databag#22] [Ideas]
Another thing I thought of that would be nice is when tapping a notification to see the new message, if it would directly open the conversation  that notification is tied to.

Currently, when tapping a notification, it takes me to the conversation list, and then I have to open the conversation to see/respond to the message.

(also just want to add that as I'm coming across and adding feature requests, that I get that you're working on this on your own and things take time to implement!)

**Comment by balzack** [comment]
Thanks for calling this out. I see this as common functionality in most other messengers. I'll add this one to the backlog, but should be pretty straightforward to implement. 

(thanks for your patience. I am working on my own on this, but hopefully I wont keep you waiting too long between releases)

**Reply by hnt3b453** [reply]
No worries! I'm just super excited you're developing Databag!
